erdgo/utils.go

```diff
@@ -0,0 +1,33 @@
+package erdgo
+
+import (
+	"encoding/hex"
+	"encoding/json"
+
+	"github.com/ElrondNetwork/elrond-go/crypto/signing"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519/singlesig"
+)
+
+// SignTransaction signs a transaction with the provided private key
+func SignTransaction(tx *Transaction, privateKey []byte) error {
```

Comment:
Please add a short, proof-of-concept unit test, with close-to real-world data. For example:

https://github.com/ElrondNetwork/elrond-sdk/blob/master/erdjs/src/simpleSigner.spec.ts

https://github.com/ElrondNetwork/elrond-sdk/blob/master/erdpy/tests/test_wallet.py#L69

Reply:
Done